controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java

```diff
@@ -353,15 +353,18 @@ public boolean isReady() {
                                           if (e != null) {
                                               log.warn("Submission for truncation for stream {} failed. Will be retried in next iteration.",
                                                       streamName);
-                                          } else {
+                                          } else if (r) {
                                               log.debug("truncation for stream {} at streamcut {} submitted.", streamName, streamcut);
+                                          } else {
+                                              log.debug("truncation for stream {} at streamcut {} rejected.", streamName, streamcut);
```

Comment:
Can we make it a warn level debug?

Reply:
no, this is harmless.. this rejection simply means the streamcut has not progressed beyond the head.. and it need not be published at warn level